### ISSUE TITLE
Current master is missing the autoconf.sh

### ISSUE THREAD

**Issue by bimargulies** [jeremie-koenig/genext2fs#1]
The autoconf.sh file is missing. The code is buildable from the release tag, but not from master.

**Comment by jeremie-koenig** [comment]
Hi, thanks for reporting.

Generated files are purposely left out of version control. With automake and autoconf installed on my system I'm able to compile with the following sequence of commands (as per the INSTALL file):

```
$ bash autogen.sh  # That script apparently uses some bash-specific constructs
$ ./configure
$ make
```

Let me know if that helps and whether documenting it somewhere else would have been helpful.

**Comment by bimargulies** [comment]
Yes, but it's the autogen.sh that I don't see in the tree anywhere.

**Comment by bimargulies** [comment]
Oh! I see, I'm supposed to download a generic one?

**Comment by jeremie-koenig** [comment]
Oh I see; you must be looking at the 'debian' branch, which is the default one for this repository. The 'master' branch does have autogen.sh. In any case, "autoreconf -i" should have the same effect as autogen.sh.

**Comment by jeremie-koenig** [comment]
No you shouldn't have to download a generic autogen.sh, "git checkout master" should do it.

**Comment by bimargulies** [comment]
Thanks. Did you see my message on the list about a missing block error.
?
